### DISCUSSION TITLE
Packages view garbled

### DISCUSSION THREAD

**Discussion by SeaMonky** [atom/atom#22900] [Q&A]
If I go to Edit > Preferences > Packages and click to expand the Community Packages or Core Packages list views, the application window gets completely garbled. Anyone else seeing this? Clicking on the list again returns it to normal.
Atom version: 1.58.0 x64
Mac OS version: 10.15.7

![atom_packages_view](https://user-images.githubusercontent.com/89536313/130825507-afa5276d-3df2-4a72-b7e4-80b1957c162c.png)

**Comment by the-j0k3r** [comment]
More information on the system specs and logs would be helpful

**Comment by icecream17** [comment]
Try this comment on another visual issue
https://github.com/atom/atom/discussions/22402#discussioncomment-832134

**Reply by the-j0k3r** [reply]
Equally as unhelpful, there is literally no actionable information to try to debug where issue is.

Neither just stating Atom version and macOS version is enough.

You need logs, system specs etc. something someone can use to test in their local setups.

The reddit thread is for VScode afaik

**Comment by icecream17** [comment]
Hmm, now that (https://github.com/atom/atom/discussions/23150) has happened it's vaguely possible there's some css issue going on

So maybe follow the css instructions there (or just reinstall atom)
